src/sql/run.py

```diff
@@ -400,6 +401,8 @@ def run(conn, sql, config, user_namespace):
                 )[0]
                 result = FakeResultProxy(cur, headers)
             else:
+                if Connection.current:
+                    statement = str(Connection.current._transpile_query((statement)))
```

Comment:
@edublancas 
Let me know if you think here is the right place to parse the original SQL clause with sqlglot

Reply:
this would transplite the user's query right? e.g.

```
%%sql
select * from table
```

why do we transpile user's queries?

I'd assume we only need to transpile queries that we run on user's behalf that they don't write explicitly (e.g., when plotting), but maybe I'm missing something

Reply:
I think let's move the transpile step [here](https://github.com/ploomber/jupysql/blob/7b9f0f17c41342c31872708dd431633c7576b09b/src/sql/store.py#L86); I think the problem is that the expected query in the test case is hardcoded; hence it might not match.

there are some changes we need to do here since now SQLStore must be aware of the current dialect (so it can pass it to SQLQuery and then, pass it to the transpile call).